TrinityCore/TrinityCore
# Core/PacketIO: Implemented SMSG_LOSS_OF_CONTROL_AURA_UPDATE

**Changes proposed:**

-  Implemented SMSG_LOSS_OF_CONTROL_AURA_UPDATE
-  

**Issues addressed:**

Closes #  (insert issue tracker number)


**Tests performed:**
- Build pass
- Use .cast t 245706 on a mob to see stun effect on your char UI
- Use .cast t 248133 on a mob to see fear effect on your char UI
(Does it build, tested in-game, etc.)


**Known issues and TODO list:** (add/remove lines as needed)
- Spells without EffectMechanic need a way to handle the EffectMechanic field from packet. - Solved
- Using .cast t 853 on a mob, make that spell don't show the right Icon - Solved
